source/documentation/3.9/rspec-core/file.Changelog.html

```diff
@@ -72,27 +73,24 @@ <h3>3.9.0 / 2019-10-07</h3>
 
 <p><a href="http://github.com/rspec/rspec-core/compare/v3.8.2...v3.9.0">Full Changelog</a></p>
 
-<p>Enhancements:</p>
-
-<ul>
```

Comment:
This is an issue 
![Capture d’écran 2020-04-08 à 12 14 59](https://user-images.githubusercontent.com/8417720/78772803-9a5a0b80-7992-11ea-91d8-6215587ac853.png)

Missing blank line in markdown:
```diff
--- a/Changelog.md
+++ b/Changelog.md
@@ -11,6 +11,7 @@ Bug Fixes:
 [Full Changelog](http://github.com/rspec/rspec-core/compare/v3.8.2...v3.9.0)

 Enhancements:
+
```

Reply:
Changed here:
- https://github.com/rspec/rspec-core/commit/0ca4fa2aa3fc7447e55facd4e50a6bc10597a383
- https://github.com/rspec/rspec-core/commit/87fe9a2b39ca17939487e954c157cd80ac3b02a1

But... We will have to release a new version to have proper doc renderer or... 

@JonRowe do you accept a "fixup" then "rebase squash" on the release commit of rspec-core 3.9.1? 🧐

Reply:
No we can't do that.

Reply:
Do we need the task to optionally accept a commit for manually setting the commit to generate from?

Reply:
> Do we need the task to optionally accept a commit for manually setting the commit to generate from?

We could but we will end up with with the changelog that contains unreleased enhancements and bug fixes.

Another very basic proposal is to add manually the proper html generated. On the next doc generation no change because we already fix it. It is not super clean. But the simplest and we have to do it only once. :)

Reply:
Branch of `v3.9.1` fix the changelog then doc gen from that branch?

Reply:
https://github.com/rspec/rspec.github.io/pull/138/commits/003568d15794b89e3324503762a8005c1e43bbbe